### PR TITLE
fix: add explicit styles for welcome tab

### LINKED ISSUE #4555: Welcome tab changes font size
### Describe the bug

If I close the last tab, then the welcome tab has a huge font-size:

capture LbBb3a_optimized

### Steps to reproduce

1. Open a new tab
2. Close it
3. **Observe huge font-size in welcome tab**

### Expected behavior

Font size stays consistent.

### Environment

- OS: Linux
- Camunda Modeler Version: v5.28.0
- Execution Platform: any
- Installed plug-ins: none

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by jarekdanielak**
Closes #4555

### Proposed Changes

Welcome tab now always looks the same, even if you open and close some diagrams.

 

### Checklist

To ensure you provided everything we need to look at your PR:

* [X] __Brief textual description__ of the changes present
* [ ] __Visual demo__ attached
* [ ] __Steps to try out__ present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [X] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

File: client/src/app/EmptyTab.less [hunk 1]
```diff
@@ -62,6 +62,10 @@
       font-weight: 14px;
     }
 
+    p {
```
Comment (nikku) [inline client/src/app/EmptyTab.less:65]: Did you know what (external) styles interfer here? Adding these styles feels like a hack to me, if we don't find the cause of the original blunders (some extension does not apply styles locally).

Comment (jarekdanielak) [reply client/src/app/EmptyTab.less:65]: I think it comes from style in `variable-outline`. Investigating... 👀

https://github.com/bpmn-io/variable-outline/blob/31340baaf9c7a9e0b5a5bb0205a6205aec29a973/lib/style.scss

Comment (nikku) [reply client/src/app/EmptyTab.less:65]: I think so, too. But either way, let's fix it at the root, not the leaf.

Comment (jarekdanielak) [reply client/src/app/EmptyTab.less:65]: Should we still merge the rest of the changes? I think it's ok to have an explicit style for empty (welcome) tab header.

Comment (nikku) [reply client/src/app/EmptyTab.less:65]: What is the benefit of adding it, if it worked previously without that change?

**Comment by jarekdanielak** [comment]
The root cause of this issue are [carbon styles user in variable outline](https://github.com/bpmn-io/variable-outline/blob/31340baaf9c7a9e0b5a5bb0205a6205aec29a973/lib/style.scss). There is no simple way to scope them without [some hacking](https://tinytip.co/tips/scss-load-css/) (thanks @marstamm), so I'll stick with explicit styles on the side of welcome page.

I realized there is no need to introduce new class as I can just reuse `tab__name` for welcome page too. Having `p font-size` defined explicitly doesn't seem like a hack for me. We want the font on this page to be this size, so it might as well say it in the styles.

**Review by nikku — APPROVED** [review]
Ready to be merged from my end (https://github.com/camunda/camunda-modeler/pull/4599#issuecomment-2413695334).

**Comment by lmbateman** [comment]
Not sure I have anything constructive to add, except maybe that the radio buttons in the Deploy popup look very odd (at least on my machine). Many thanks to Nico for looping me in and helping me to understand the line height issues.

**Comment by nikku** [comment]
@lmbateman Let's verify on [tomorrows nightly](https://downloads.camunda.cloud/release/camunda-modeler/nightly/) if they look odd still. Should be fixed with this PR.

**Comment by barmac** [comment]
Still reproducible:
image

**Comment by nikku** [comment]
Yea. Was merged to `main`, pending integration into `develop`, which is done now.

Fixed in tomorrows nightly :upside_down_face:, or on `develop`:

image

**Comment by barmac** [comment]
Confirmed:
image

**Pull request merged**